conda/recipes/versions.yaml

```diff
@@ -90,7 +90,7 @@ networkx_version:
 nodejs_version:
   - '>=12'
 numba_version:
-  - '>=0.49,<0.51'
+  - '>=0.49'
```

Comment:
Do we want to allow Numba 0.51 or do we want to require it?

Reply:
I think we should just allow it